modules/core/src/effects/lighting/suncalc.js

```diff
@@ -42,7 +42,7 @@ export function getSunlightDirection(timestamp, latitude, longitude) {
   const azimuthN = azimuth + Math.PI;
 
   // solar position to light direction
-  return [-Math.sin(azimuthN), -Math.cos(azimuthN), -Math.sin(altitude)];
+  return [-Math.sin(azimuthN), Math.cos(azimuthN), -Math.sin(altitude)];
```

Comment:
unit test is not impacted by this change?

Reply:
I think only `getSolarPosition` is tested.